lib/buffer.js

```diff
@@ -1,13 +1,18 @@
+/* eslint-disable no-class-assign */
 /* eslint-disable require-buffer */
 'use strict';
 
 const binding = process.binding('buffer');
 const { isArrayBuffer } = process.binding('util');
+const { deprecate } = require('internal/util');
 const bindingObj = {};
 
-class FastBuffer extends Uint8Array {}
+class Buffer extends Uint8Array {}
+
+// to avoid code churn...
+const FastBuffer = Buffer;
+Buffer = DummyBuffer;
```

Comment:
I think I’d be okay with the code churn created by resolving these if it helps clarify what each name here refers to?